src/Symfony/Component/HttpClient/composer.json

```diff
@@ -23,7 +23,7 @@
     "require": {
         "php": "^7.2.5",
         "psr/log": "^1.0",
-        "symfony/http-client-contracts": "^1.1.8|^2",
+        "symfony/http-client-contracts": "^2.1.1",
```

Comment:
tag to be added on symfony/contracts and symfony/http-client-contracts after merge